tasks/text_task/chunker.yaml

```diff
@@ -0,0 +1,4 @@
+task: chunker
```

Comment:
Thanks, @calpt and no problem, I'll fix it tomorrow! Let's hold the merge until then.

Reply:
@calpt now that I am looking closer at all the task names both in terms of YAML file naming and display names, isn't a noun `chunk` a better match for YAML file name rather than a verb `chunking`, just like `qa`, `sentiment`, `commonsense` etc. However,  just like Question Answering, Sentiment Analysis, etc I'll name the task (display name) Phrase Chunking. LMK

Reply:
I wanted to try out the notebook with a new zip file `chunk-conll2003.zip` and task renamed to `chunk`. I thought I only needed to change the name of the task in the zip file (adapter_config.json) and adjust the name of the model adapter passed to model inference call. Although inference call worked I got a response that was totally off. Interested what is going on @calpt ?!

Reply:
I have a preview of changes on another [branch](https://github.com/vblagoje/Hub/commit/870b596ff8fd2edbd4b903ace3cb8a66aa754754) The notebook now works. I just retrained the model with a new adapter name. LMK @calpt

Reply:
> isn't a noun chunk a better match for YAML file name rather than a verb chunking

Yes, that's also fine. While your're at it, maybe quickly also change `POS Tagger` to `POS Tagging` (sorry, didn't pay attention to this in your last PR). Thanks!

>  I thought I only needed to change the name of the task in the zip file (adapter_config.json) and adjust the name of the model adapter passed to model inference call

The name in the json file is used to identify the correct weights when loading and to insert them in the correct places in the model. So, when changing the name in the json, the name must also be changed in the pickled weights file. Something like this should do the trick (using `load_as`):

``
model.load_adapter("/load/path", load_as="chunk")
model.save_adapter("/save/path", "chunk")
``

and as sidenote: the name in the json doesn't have to match the task name in the yaml, so just renaming the task in the yaml file would've been enough.

> I have a preview of changes on another branch The notebook now works. I just retrained the model with a new adapter name.

Looks good, just add it to this PR.

Reply:
Thanks for the weights tips @calpt . If we are changing `POS Tagger` to `POS Tagging` I used the opportunity to set the default_version to 1 in adapters/vblagoje/bert-base-uncased-conll2003_pfeiffer.yaml. Please look at that change as well. Next time I'll just add commits to PR rather than force push. LMK if I should adjust anything else.

Reply:
> Next time I'll just add commits to PR rather than force push.

I'll squash them anyway during merging, so this really doesn't make a difference in the end :)

As we somehow got back to the POS tagging adapter already, the very last little step to perfection would be to also remove the citation there (as @arueckle mentioned above for this adapter). That's definitely the last thing from my side, rest looks very good!

Reply:
No problem @calpt Done

Reply:
@vblagoje Thanks! Merging now.